I've edited the Add Product, Edit Product, Search Product

Fr3shm3n opened the pull request.

Fr3shm3n committed:
Changes

Fr3shm3n committed:
Imports for Edit Product Component

Changes to Add, Edit, Search Product.

Fr3shm3n committed:
Merge remote-tracking branch 'refs/remotes/vlw0052/master'

# Conflicts:
#	src/app/products/add-product.component.ts
#	src/app/products/edit-product.component.ts
#	src/app/routes.ts
#	src/app/services/product.service.ts

Fr3shm3n committed:
Changed Product to reflect changes made to Discount

leonwatson2 reviewed (commented):
Everything looks good just change your routes to only work with the correct type of associates.
https://youtu.be/yqzaP6yT15k?t=2m33s
You can watch a little explanation here.

Fr3shm3n committed:
Slaved away at the computer until I got ADD, EDIT, SEARCH functioning completely

Went through all the components composing Discount and adjusted it to
work with Products. Everything seems to function, I've added items,
tried adding the same item again and an error message displays, I've
changed an item name. I haven't used the search item since the inventory
is so small. But I think we are good.
